zeus/core.py

```diff
@@ -1200,7 +1200,6 @@ def gamma_decode_to_range_ballot(encoded, candidates_and_points):
     permutation = to_absolute_answers(selection, nr_candidates)
     ballot = {}
     counts = {}
-    valid = False
```

Comment:
This is correct but confusing, I would leave that `valid = False` here (and perhaps remove the latter lines: do just `break` instead of `valid = False` then `break`).